emf/src/main/java/io/lionweb/lioncore/java/emf/mapping/ConceptsToEClassesMapping.java

```diff
@@ -121,6 +136,28 @@ public void registerMapping(Interface iface, EClass eClass) {
     interfacesToEClasses.put(iface, eClass);
   }
 
+  public void registerMapping(Language language, EPackage ePackage) {
+    for (LanguageEntity entity : language.getElements()) {
```

Comment:
Should we check if this mapping already exists and in that case just return?
Should we check for non-conflict mapping?